charts/argo-cd/values.yaml

```diff
@@ -718,7 +730,7 @@ redis:
       # -- Metrics service port
       servicePort: 9121
       # -- Metrics service port name
-      portName: http-metrics
+      portName: metrics
```

Comment:
Are you sure that we need to rename the service ports from "http-metrics" to "metrics". There were PRs which did the oposite of your work in the past and also referenced that this is necessary for istio:
- https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/
- https://github.com/argoproj/argo-helm/pull/889
- https://github.com/argoproj/argo-helm/pull/956
- https://github.com/argoproj/argo-helm/pull/236

As I don't use istio, I cannot confirm what works better.

Reply:
Hi @mkilchhofer 

Thank You, greetings back from Zurich to Bern.  Currently the hard coded value for the metrics service is `metrics`. I don't know what you prefer, setting an istio compatible default, which would be `http-metrics` or leave the default at its current value `metrics`. 
To be consistent I'd use the same value everywhere. Imo, because non-istio-users don't really mind, I'd change it to `http-metrics` and adjusted the PR accordingly.